frontend/src/components/worksheets/items/MarkdownItem.js

```diff
@@ -12,6 +14,20 @@ import TextEditorItem from './TextEditorItem';
 import { createAlertText } from '../../../util/worksheet_utils';
 import Tooltip from '@material-ui/core/Tooltip';
 import { addItems } from '../../../util/apiWrapper';
+
+marked.setOptions({
+    renderer: new marked.Renderer(),
+    gfm: true,
+    pedantic: false,
+    tables: true,
+    breaks: true,
+    smartLists: true,
+    smartypants: true,
+    highlight: function(code) {
+        return hljs.highlightAuto(code).value;
```

Comment:
Does this function end up reading the "```python" at the beginning of the code block? It appears that this function, rather, just auto-detects language based on the content. Do you think it'd be possible to change this to do the former, and if no language is specified, just not do any highlighting?

Reply:
For the screenshots, the source code is the following
<pre>
```python
x = [1 for i in range(0, 100)]
# This program prints Hello, world!
print('Hello, world!')
```

```js
componentDidMount() {
    this.processMathJax();
}
componentDidUpdate() {
    this.processMathJax();
    if (this.props.focused) this.capture_keys();
}
```
</pre>

As for the language, when I was reading the hightlight.js documentation, it stated that the language is provided through the class attribute, which the original already does. I believe that the package will always try to automatically detect the language of the code block if no language is provided through the class attribute unless the block is given a "plaintext" class.

Reply:
I think it should be possible to default to "plaintext" as the provided language prop if none is provided by the user. Another possible solution would be to edit the documentation so that ```plaintext would denote the start of a block of plaintext code without highlighting.

Reply:
Pushed a new update that disables automatic detection and has the desired functionality (highlights with the language given in code block, does not highlight if none provided).